drivers/SmartThings/philips-hue/profiles/legacy-color.yml

```diff
@@ -0,0 +1,20 @@
+name: legacy-color
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+    config:
+      values:
+        - key: "level.value"
+          range: [ 1, 100 ]
+  - id: colorControl
+    version: 1
+  - id: samsungim.hueSyncMode
```

Comment:
Where does this live?

Reply:
No clue. HQ owns it. It's in Prod though.